### PR TITLE
Update README and add few more notes

### PR DESCRIPTION
**Pull request by bastilian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,15 +57,34 @@ The frontend itself can be started with `npm run start` once all dependencies ar
 
 ## Code Notes
 
-* This project uses [Patternfly](https://github.com/patternfly/patternfly-react) components and should be preferred.
-* The [insights-frontend-components](https://www.npmjs.com/package/@red-hat-insights/insights-frontend-components) package is included to provide components shared across the Insights Platform.
-* Header and sidebar are provided by  [insights-proxy](https://github.com/RedHatInsights/insights-chrome)
-* "Inventory Components" (like the SystemsTable) are loaded **via** the chrome.
+### Used technologies
```
Comment (Victoremepunto) [inline README.md:60]: Technology stack ?

File: README.md [hunk 2]
```diff
@@ -55,15 +57,34 @@ The frontend itself can be started with `npm run start` once all dependencies ar
...
+* Prefer functional components and [hooks](https://reactjs.org/docs/hooks-intro.html) over class components
+* The [insights-frontend-components](https://www.npmjs.com/package/@red-hat-insights/insights-frontend-components) package is included to provide components shared across the Insights Platform.
+* [Insights Chrome](https://github.com/RedHatInsights/insights-chrome) which provides header and sidebar, as well as authentication and related functions, which is injected/included via
+[insights-proxy](https://github.com/RedHatInsights/insights-chrome)
+
+#### Inventory Components
+
+Some components are "hot loaded" via  the chrome. These are known as "Inventory Components" and are used for systems tables and systems details. The source for these components can be found in [insights-frontend-components](https://github.com/RedHatInsights/frontend-components/tree/master/packages/inventory#readme). With the frontend-components repository resides the [inventory-compliance](https://github.com/RedHatInsights/frontend-components/tree/master/packages/inventory-compliance) package, which implements wrapper components for these inventory components to add compliance specific behaviour.
```
Comment (Victoremepunto) [inline README.md:87]: Link to "the chrome" project, just to emphasize better, or name it as "Insights Chrome" just as it is listed before.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
 [Insights Proxy](https://github.com/RedHatInsights/insights-proxy/blob/master/scripts/patch-etc-hosts.sh) provides a script to patch the `/etc/hosts` file for this purpose.
 
-## Running in containers
+### In containers (recommended)
 
 To run the container setup either Podman or Docker and their compose commands can be used to
 
 ```shell
-  $ touch .env  # Can be used to enable a local backend and other services (see .env.example)
+  $ cp .env.example .env # Can be used to enable a local backend and other services
```
Comment (bastilian) [inline README.md:22]: With an empty `.env` `podman-compose up` will fail to start properly.

File: README.md (hunk 1 above)
Comment (bastilian) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (bastilian) [reply README.md]: Done.

**Review by akofink — APPROVED** [review]

**Pull request merged**